community/i3-gaps/patches

```diff
@@ -0,0 +1 @@
+../i3/patches
```

Comment:
This should be a folder of the actual patches dir.

Reply:
These are all symlinks. The previous package utilised symlinks, so I was under the impression that they were supported or allowed. I'll replace them with duplicated files from `i3` if you wish.